templates/hotel-booking/layout/footer.js

```diff
@@ -0,0 +1,6 @@
+import { html } from '/src/main.js';
```

Comment:
This file is an old version.

Reply:
I blame rebase, I did it in master before checking out to a new branch from master :D will fix